readme.md

```diff
@@ -1,72 +1,42 @@
-```
-
-#### Option A - ....via 'GIT Clone' ####
-Clone the Nightly GIT repository into a folder of your choice:
-
-```
-git clone https://github.com/retrospect-addon/plugin.video.retrospect.git
-```
-
-The cloned GIT repository should contain the folders mentioned above. Now either `copy` or `symlink` (`junction` on Windows) each those folders into the Kodi Add-ons folder. I would suggest using symlinks or junctions so changes from a `git pull` are automatically available in Kodi. 
-
-#### Option B - ....via 'Full Zip Download' ####
-Download the complete GIT repo and extract it into the Kodi Add-on folder.
+There are two ways to install Retrospect, depending on what version of Kodi you are using:
 
-#### Caution!
-Be aware that if you install it using *Option A* or *Option B* Kodi will **disable** the add-on by default and it will **not install any of the dependencies**. So you need to enable it manually and install all the dependencies by hand. Starting from Kodi Leia this is easier as it has a "View Dependencies" option of add-ons.   
+### Kodi Leia and later
+Let's start with the most recent version of Kodi: Leia (18.0) and newer version such as Matrix (19.0). Users running these version of Kodi can easy install Retrospect via the Kodi repository. Follow these steps (keep in mind they might slightly differ depending on your skin):
 
+1. Go to Add-ons
+1. Add-on Browser
+1. Install from repository
+1. Video Add-ons
+1. Find _Retrospect_ and select it
+1. Hit _Install_ to install Retrospect
+1. Agree to install any dependencies that are needed
 
-### 3 - Updating Retrospect ###
+Make sure to enable the _Auto-Update_ option for Retrospect to automatically receive new updates.
 
-#### Option A - ....via 'GIT Clone' ####
-Pull latest changes into your clone (located in the Kodi Add-on folder). After that remove all existing `*.pyc` and `*.pyo` files within the Retrospect folders (**don't skip this**).
+![alt text](./plugin.video.retrospect/resources/media/retrospect01.jpg "The Retrospect information screen")
 
-#### Option B - ....via 'Full Zip Download' ####
-Download the complete GIT repo. Remove all existing Retrospect folders (**don't skip this**) and extract the new ones it into the Kodi Add-on folder.
+### Kodi Krypton    
+If you are running Kodi Krypton the only way to install Retrospect is to install it from a release zip file. This is because Retrospect is not in the Kodi repository for Krypton. Follow these steps (keep in mind they might slightly differ depending on your skin):
 
-#### Finalizing the update ####
-In both situation run Retrospect at least once before accessing the Retrospect add-on settings. The initial run might take longer than usual, as Retrospect is initialising some stuff and downloads artwork.
+1. Download the latest release zip file from here: [https://github.com/retrospect-addon/plugin.video.retrospect/releases](https://github.com/retrospect-addon/plugin.video.retrospect/releases)
+1. Open Kodi
```

Comment:
This will not work, you have to indent the sub-listing.

Reply:
I don't need any sub-listing here.